umm-spec-lib/test/cmr/umm_spec/test/migration/version_migration.clj

```diff
@@ -1133,6 +1133,34 @@
            collection-contact-persons))))
 
 (deftest migrate-1_9-up-to-1_10
+  (is (= [{:PrecisionOfSeconds "3"
```

Comment:
Put this in a  `(testing ... )` block and the `is` below in a testing block to organize these.